src/shared/discover-image.c

```diff
@@ -201,29 +206,44 @@ static int image_new(
         return 0;
 }
 
-static int extract_pretty(const char *path, const char *suffix, char **ret) {
+static const char *get_image_class_suffix(ImageClass c) {
+        if (c < 0)
+                return NULL;
+
+        assert(c < _IMAGE_CLASS_MAX);
+        return image_class_suffix[c];
+}
+
```

Comment:
Why not just add a table? Shouldn't be too much extra work and then just have image_class_suffix_to_string()

Reply:
I have no idea why i didn't do this. must be old age.